### PR TITLE
Added swiftDeveloperExtras to Dangerfile 

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] Dangerfile
- [x] .gitignore
- [x] More documentation

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Error</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>Developer Specific file shouldn&#39;t be changed</td>
    </tr>
    
  </tbody>
</table>
  

  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

File: Artsy/View_Controllers/App_Navigation/ARTopMenuViewController+SwiftDeveloperExtras.swift [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+// written in or compatible with Objective-C
+
+// It is put in gitignore as the current state of what you see now,
+// any changes you make will be erased when you switch branches.
+
+
+// If you're seeing changes to this file, run this two commands
+/*
+ git checkout Artsy/View_Controllers/App_Navigation/ARTopMenuViewController+DeveloperExtras.m
+ git update-index --assume-unchanged Artsy/View_Controllers/App_Navigation/ARTopMenuViewController+SwiftDeveloperExtras.m
```
Comment (alloy) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuViewController+SwiftDeveloperExtras.swift]: These filenames aren’t correct.

**Comment by alloy** [comment]
> Danger: Ignore "Developer Specific file shouldn't be changed"

**Comment by sarahscott** [comment]
Hmm how should I deal with this danger error?

**Comment by orta** [comment]
Sometimes you just have to tell Danger that it's OK, and you know what you're doing.

**Comment by alloy** [comment]
Danger’s README mentions commenting that text that I commented, but I guess it has to be in a the original issue.

**Comment by orta** [comment]
Good point @alloy make an issue

**Pull request merged**